package.json

```diff
@@ -101,16 +101,20 @@
     "@embroider/util": "^1.2.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
+    "ember-auto-import": "^2.4.0",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-htmlbars": "^6.0.1",
     "ember-cli-typescript": "^4.2.1",
-    "ember-element-helper": "^0.6.0",
+    "ember-element-helper": "^0.6.1",
     "ember-get-config": "^2.1.1",
     "ember-maybe-in-element": "^2.0.3",
-    "ember-modifier": "^3.2.7",
-    "ember-style-modifier": "^0.8.0 || ^1.0.0",
+    "ember-modifier": "^3.2.7 || ^4.0.0",
+    "ember-style-modifier": "^0.8.0 || ^1.0.0 || ^2.0.0 || ^3.0.0",
     "ember-truth-helpers": "^2.1.0 || ^3.0.0"
   },
+  "peerDependencies": {
+    "@ember/string": "^3.0.1"
```

Comment:
I don't think you need having `@ember/string` as a peer dependency here. It should only be needed if the addon imports from `@ember/string` itself. Adding it to `devDependencies` should be enough if it only depends on another addon (here: `ember-style-modifier`), which requires that package. Adding it to `devDependencies` makes it available for the test app, which should be enough.

The difference is important if `ember-style-modifier` drops the peer dependency. If you have it declared as a peer dependency here as well, consumers would still need to install it even if it is not used anymore.

Reply:
@jelhan sry my delay... thats right, i have removed the peerDependency